### ISSUE TITLE
Error on ReadMe reffering to colors

### ISSUE THREAD

**Issue by japarico** [do9core/roulette#17]
In your ReadMe, there is a reference to color which says.

colorBuilder: Colors.blue,

But this will generate an error.

**Comment by do9core** [comment]
Thanks for pointing this out.  
This is a mistake in the document.

The proper way is :

```dart
colorBuilder: (index) => Colors.blue, // Use your own color
```

I'll fix this as quickly as possible.

**Comment by do9core** [comment]
Fixed in 0.1.3+1
